components/automate-ui/src/app/components/delete-object-modal/delete-object-modal.component.html

```diff
@@ -1,6 +1,6 @@
 <chef-modal [visible]="visible" (close)="closeEvent()">
-  <h2 slot="title">Delete {{objectNoun | titlecase}}?</h2>
-  <div>
+  <h2 slot="title">{{ObjectAction}} {{objectNoun | titlecase}}?</h2>
+  <div *ngIf="!nodeCredObject">
```

Comment:
Thanks for adding `ObjectAction`! I've got one more ask: let's drop `nodeCredObject` from this component and use a pipe [like this](https://github.com/chef/automate/blob/master/components/automate-ui/src/app/pipes/pluralize.pipe.ts) instead. I think this would work (though I haven't tested it):
```
export class ContinuousPipe implements PipeTransform {

  transform(value: string): string {
    const lastIndex = value.length - 1
    if (value) {
      if (value.charAt(lastIndex) == 'e') {
        return value.slice(0, lastChar - 1) + 'ing';
      } else {
        return value + 'ing';
      }
    }
    return value;
  }
}
```

Then we use `{{ ObjectAction | continuous }}` in the template

Reply:
I have implemented pipe the same as you suggested, please have a look.